RecoTracker/MkFit/test/dumpMkFitGeometryPhase2.py

```diff
@@ -0,0 +1,57 @@
+import FWCore.ParameterSet.Config as cms
+
+### from Configuration.Eras.Era_Run3_cff import Run3
+### from Configuration.Eras.Era_Phase2C17I13M9_cff import Phase2C17I13M9
+### from Configuration.ProcessModifiers.trackingMkFit_cff import trackingMkFit
+
+### process = cms.Process('DUMP',Run3,trackingMkFit)
+process = cms.Process('DUMP')
+
+# import of standard configurations
+###process.load('Configuration.StandardSequences.Services_cff')
+###process.load('FWCore.MessageService.MessageLogger_cfi')
+###process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+###process.load('Configuration.StandardSequences.MagneticField_cff')
+###process.load('Configuration.StandardSequences.Reconstruction_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag = autoCond['run2_mc']
```

Comment:
```suggestion
process.GlobalTag.globaltag = autoCond['phase2_realistic']
```

this doens't look appropriate for a file called `dumpMkFitGeometryPhase2.py`.
Is a GlobalTag even needed here? If yes, the above looks more appropriate.

Reply:
To be honest, I welded this and phase1 config from Fireworks/Geometry reco-geometry dumper and would appreciate if somebody helps me clean them both up and only pull in what is needed.
I can of course also just make the above change.

Reply:
For starters, I'd copy the structure of the Run-3 one here:
https://github.com/cms-sw/cmssw/blob/80ed78b34c4d4bf5de1de32917da5c85b6e40855/RecoTracker/MkFit/test/dumpMkFitGeometry.py#L1-L38

changing:
   * the era from `Run3` to be `Phase2C17I13M9` (see [upgradeWorkflowComponents.py](https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L2115-L2121))
   * `process.load('Configuration.StandardSequences.GeometryRecoDB_cff')` with `process.load('Configuration.Geometry.GeometryExtended2026D88Reco_cff')`
   * the `GlobalTag`  as suggested above.
   
In addition perhaps this new addition can be tested adding a line here:

https://github.com/cms-sw/cmssw/blob/80ed78b34c4d4bf5de1de32917da5c85b6e40855/RecoTracker/MkFit/test/testDumpMkFitGeometry.sh#L10

Reply:
Thank you ... done in 529b93b